### DISCUSSION TITLE
Lifecycle categories

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#375] [Ideas] [answered]
I would like to suggest to split the "complete" category in lifecycle management into two seperate categories:
- Prototype
- In production

We have some opportunities that have "stranded" (for now) in protoype phase because they are for example awaiting new policy or laws. They are not abandoned bot not exactly complete either.

**Comment by techsmyth** [comment, marked as answer]
Actually the goal is to take it further and allow the selection of the lifecycle template to use as part of the challenge / opportunity creation page.

And then beyond that to allow ecoverse admins to also upload their own template. This is for me part of the wider Templates design: https://github.com/alkem-io/alkemio/issues/314

Feel free to also put some comments in there...

In the near term feel free to suggest an updated lifecycle template if you are feeling technical ;) The default definition is in the following file:
https://github.com/alkem-io/server/blob/develop/src/domain/challenge/challenge/challenge.lifecycle.config.default.ts
and you can visualize the contents using: https://xstate.js.org/viz/ (you do need to make sure you copy / paste only the definition over).

This is a space with a lot to explore.

**Reply by FloriSpoelstra** [reply]
Very nice! I might try and experiment a bit with this. I have some ideas about lifecycle management for our challenges/opportunities.
